static/icons/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "TBA",
```

Comment:
Actually we already have a manifest.json here: https://github.com/the-blue-alliance/the-blue-alliance/blob/master/static/manifest.json

Perhaps we should use that one instead and make sure you point to /manifest.json instead of /icons/manifest.json